src/components/ui/select/select.component.tsx

```diff
@@ -72,6 +72,8 @@ export interface SelectProps extends StyledComponentProps, TouchableOpacityProps
   controlStyle?: StyleProp<ViewStyle>;
   icon?: IconProp;
   onSelect: (option: SelectOption, event?: GestureResponderEvent) => void;
+  onFocus: () => void;
+  onBlur: () => void;
```

Comment:
Thanks for adding these. onFocus and onBlur should be optional props.

<img width="671" alt="Screenshot 2020-02-10 at 23 20 32" src="https://user-images.githubusercontent.com/899175/74199408-f8e95e80-4c5b-11ea-8476-de5c40b3ec28.png">

Reply:
how could I do that :(

Reply:
no worries. it happens to the best of us 😅